.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+---
+stages:
```

Comment:
nit: Does a Distelli module need a gitlab file?

Reply:
@danielparks it looks like this was added as part of your "Rename distelli_agent to pipelines". Is this file used? If not, I think @glennsarti has a valid point :).

Reply:
Sorry, I missed this. :) It's from PDK, so it will be recreated next update. You could add it to .gitignore, though.